### ISSUE TITLE
Keyboard navigation (up/down arrows) doesn't work in vertical tab view

### ISSUE THREAD

**Issue by MadhaviSeelam** [brave/brave-browser#30018]
## Description 

After enabling vertical tabs and adding few tabs, scrolling thru tabs with keyboard up/down arrow keys didn't work.  Keyboard navigation didn't work in horizontal view either.  Ctrl + tab, however, just fine in both horizontal tab and vertical tab views.  

## Steps to Reproduce

   1. Install 1.51.105
   2. launch Brave
   3. add few tabs in horizontal tab view
   4. enable vertical tabs 
   5. scroll thru the tabs with keyboard up/down arrow keys
   6. able scroll thru with ctrl + tab in both vertical and horizontal tab views

## Actual result:

scrolling through the tabs in vertical tab view with keyboard up/down arrow keys didn't work (as this might be an upstream issue since this didn't work in Chrome and Edge either.)

## Expected result:
Keyboard navigation should work 

## Reproduces how often: 

Easily

## Brave version (brave://version info)

```
Brave | 1.51.105 Chromium: 112.0.5615.165 (Official Build) beta (64-bit)
-- | --
Revision | c262f36e6b1d711ee42d4fbe1343b49960593f18-refs/branch-heads/5615@{#1297}
OS | Windows 11 Version 22H2 (Build 22621.1555)
```

## Version/Channel Information:

- Can you reproduce this issue with the current release? N/A
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards? 
- Is the issue reproducible on the latest version of Chrome? 

## Miscellaneous Information:
@rebron

**Comment by ajayseeker** [comment]
Hi @MadhaviSeelam and @rebron, actually UpArrow and DownArrow keys also have this functionality of scrolling up and down the web page. So, probably changing that functionality to switching tabs in vertical tab view can be discomforting for the users who use UpArrow and DownArrow keys for scrolling.

If my point makes sense, I would rather like to propose we do the following:
1. In Mac :- make `cmd+option+UpArrow` selects tab above in vertical view and similarly `cmd+option+DownArrow` selects the tab below. This can be easily achieved by adding a shortcut in `src/chrome/browser/global_keyboard_shortcuts_mac.mm` and I've tested it to be working.
2. In Windows (and other Os) :- make `control + UpArrow` select the tab above and `control + DownArrow` select the tab below. I guess this can be achieved by adding an Accelerator I believe, couldn't test it as I don't have windows machine handy.

**Comment by sangwoo108** [comment]
I didn't know that arrow key could be used for tab switching. It might be something related to accessibility or focus traversal with tab key. I tried to set focus on tabs but upstream seems to have changed that tabs can't be focused in focus traversal. I think we can close this.

**Comment by ajayseeker** [comment]
okay! But won't being able to maneuver between tabs in Vertical Tab view using some shortcut involving Up and Down arrow a good idea?
Especially in Mac, when we maneuver in horizontal tab view using `cmd + option + {Left/RIght}` gives a sense of moving left and right in tab view by pressing Left and Right arrow keys. If we could move Up and Down in Vertical Tab view using `cmd + option + {Up/Down}`, instead of `cmd + option + {Left/Right}` it will be a better user experience, isn't it? 

@sangwoo108 what do you say?

**Comment by sangwoo108** [comment]
As we already have tab navigation shortcut like `control pg-up/down` or `cmd shift []`. Also users can add their own shortcut in settings page, I don't feel strong about adding new default shortcut.
